src/components/shared/Entity/EndpointConfig.tsx

```diff
@@ -11,11 +14,17 @@ interface Props {
 function EndpointConfig({ connectorImage }: Props) {
     const { connectorTag, error } = useConnectorTag(connectorImage);
 
+    const draftId = useZustandStore<
+        EditorStoreState<DraftSpecQuery>,
+        EditorStoreState<DraftSpecQuery>['id']
+    >((state) => state.id);
+
     if (error) {
         return <Error error={error} />;
     } else if (connectorTag) {
         return (
             <WrapperWithHeader
+                forceClose={draftId !== null}
```

Comment:
Go ahead and force close the endpoint config section once we have a draft ID (which means we have a catalog to edit)